apps/vanilla/src/pages/bookCatalog/index.html

```diff
@@ -0,0 +1,120 @@
+
+							<span class="book__title typography__body_small">Refactoring UI</span>
+						</li>
+						<li class="book__item">
+							<img class="book__image" src="./image/image-1.png" alt="Master the Skill of Debugging CSS">
+
+							<span class="book__title typography__body_small">Master the Skill of Debugging CSS</span>
+						</li>
+						<li class="book__item">
+							<img class="book__image" src="./image/image-9.png" alt="Blue Period">
+
+							<span class="book__title typography__body_small">Blue Period</span>
+						</li>
+
+					</ul>
+				</div>
+				<div class="books__group">
+					<h2 class="books__header typography__subtitle">
+						1980 year
+					</h2>
+					<ul class="book">
+						<li class="book__item">
+							<img class="book__image" src="./image/image-10.png" alt="Yokohama Kaidashi Kikou">
+
+							<span class="book__title typography__body_small">Yokohama Kaidashi Kikou</span>
+						</li>
+						<li class="book__item">
+							<img class="book__image" src="./image/image-4.png" alt="Learning JavaScript Design Patterns">
+
+							<span class="book__title typography__body_small">Learning JavaScript Design Patterns</span>
+						</li>
+						<li class="book__item">
+							<img class="book__image" src="./image/image-11.png" alt="Yotsuba to!">
+
+							<span class="book__title typography__body_small">Yotsuba to!</span>
+						</li>
+						<li class="book__item">
+							<img class="book__image" src="./image/image-5.png" alt="Domain-Driven Design">
+
+							<span class="book__title typography__body_small">Domain-Driven Design</span>
+						</li>
+						<li class="book__item">
+							<img class="book__image" src="./image/image-6.png" alt="Implementing Domain-Driven">
+
+							<span class="book__title typography__body_small">Implementing Domain-Driven</span>
+						</li>
+						<li class="book__item">
+							<img class="book__image" src="./image/image-8.png" alt="7 Навыков высокэффективных">
+
+							<span class="book__title typography__body_small">7 Навыков высокэффективных</span>
+						</li>
+					</ul>
+				</div>
+			</section>
+
+			<div class="content__sidebar-wrapper">
+				<div class="content__sidebar sidebar">
+					<span class="sidebar__title typography__subtitle">Filters</span>
+					<form class="sidebar__inputs">
+						<input class="sidebar__search typography__body" type="search" name="search"
+							placeholder="e.g. “Refactoring”">
+
+						<select class="sidebar__select typography__body" name="sort-select">
+							<option value="" selected disabled>Choose sort field</option>
+							<option value="Length">Length</option>
+							<option value="Short">Short</option>
+						</select>
+
+
+						<label class="sidebar__checkbox-container typography__body">
+							<input class="sidebar__checkbox" type="checkbox" name="Show comics">
+							Show comics
+							<span class="sidebar__checkbox-mark"></span>
+						</label>
+					</form>
+				</div>
+			</div>
+		</main>
+
+		<footer class="main-footer">
```

Comment:
The `main` prefix looks redundant for footer because because he is alone and there are no `additional`

```suggestion
		<footer class="footer">
```